### PR TITLE
urlscan: Update to 1.0.1

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Changes can be found [here](https://github.com/firecat53/urlscan/releases)
- Added `homepage` key to `package.yml` (Part of #411)

**Test Plan**

- Ran it successufully on test file
- Verified the added homepage loads information about the package

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by malfisya** [comment]
There is newer version at 1.0.1, but it uses 'pyproject.toml' instead of 'setup.py'. How do i point '%python3_install' macro to detect that file instead?

**Comment by joebonrichie** [comment]
> There is newer version at 1.0.1, but it uses `pyproject.toml` instead of `setup.py`. How do i point `%python3_install` macro to detect that file instead?

%python3_install should do automatically, it's basically if setup.py exists use that otherwise use pyproject.toml. If you want to force it to use pyproject.toml rm the setup.py file.

**Comment by malfisya** [comment]
hmmh, there is no `setup.py` file on the tarball, yet it doesn't detect it automatically. I will try this again later

**Comment by joebonrichie** [comment]
oh for pyproject.toml files you typically need `python-build`, `python-packaging`, `python-wheel` and `python-installer` as builddeps. Some pyproject.tomls will also use custom build systems such as `python-flit` or `python-poetry`

**Comment by malfisya** [comment]
Now I got this [error](https://bpa.st/DMMQ).

**Comment by joebonrichie** [comment]
You probably also need `python-setuptools-scm` in the builddeps as well. With python you just have to sort of iterate over these problems currently as we don't any have automatic dependency solving for python.

**Comment by malfisya** [comment]
Got it. Thanks for walking me through this. I usually just move on if its not simple to build, I will try finish this😄

**Comment by malfisya** [comment]
changed the source from github to pypi. it really nags on some metadata only available on pypi source. it builds! also taken some inspiration from arch PKGBUILD

File: packages/u/urlscan/package.yml [hunk 1]
```diff
@@ -1,15 +1,20 @@
 name       : urlscan
-version    : 0.9.4
-release    : 7
+version    : 1.0.1
+release    : 8
 source     :
-    - https://github.com/firecat53/urlscan/archive/0.9.4.tar.gz : fc76c3c8a852f1fbdde69faa336184b7f060b33e8e4573d6ffa8790ed6f429b6
+    - https://files.pythonhosted.org/packages/b7/7d/984994a32b261cc3a72d3bfb0b8c0de4f786682128dc659a5c5e02dfc48c/urlscan-1.0.1.tar.gz: e0ec986e5aa2da57dd2face8692116d80af173d4eb56a78e4fd881731113307f
```
Comment (joebonrichie) [inline packages/u/urlscan/package.yml:5]: Can you change the source to `https://files.pythonhosted.org/packages/source/u/urlscan/urlscan-1.0.1.tar.gz` <- more readable and doesn't break `go-task bump` formatting.

File: packages/u/urlscan/package.yml [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+homepage   : https://github.com/firecat53/urlscan
 license    : GPL-2.0-or-later
 component  : network.mail
 summary    : Mutt and terminal url selector (similar to urlview)
 description: |
     Urlscan is a small program that is designed to integrate with the "mutt" mailreader to allow you to easily launch a Web browser for URLs contained in email messages. It is a replacement for the "urlview" program.
 builddeps  :
-    - pkgconfig(python3)
+    - python-build
```
Comment (joebonrichie) [inline packages/u/urlscan/package.yml:13]: alphabetized builddeps pkgconfig,0-9,A-z,a-z

**Review by joebonrichie — CHANGES_REQUESTED** [review]

File: packages/u/urlscan/package.yml (hunk 2 above)
Comment (malfisya) [reply packages/u/urlscan/package.yml:13]: ok

File: packages/u/urlscan/package.yml (hunk 1 above)
Comment (malfisya) [reply packages/u/urlscan/package.yml:5]: ok

**Review by joebonrichie — APPROVED** [review]

**Pull request merged**